--- origsrc/xforms-1.2.4/doc/Makefile.am	2013-12-19 06:21:15.000000000 -0500
+++ src/xforms-1.2.4/doc/Makefile.am	2017-06-10 21:51:32.881871500 -0400
@@ -92,7 +92,7 @@ uninstall-local:
 	cp -r xforms_images xforms.html
 
 .texi.pdf:
-	if [ $(TEXI2DVI_NEEDS_PDF) = "yes" ]; then \
+	if [ x$(TEXI2DVI_NEEDS_PDF) = "xyes" ]; then \
 		$(MAKE) -C xforms_images create_pdf_files; \
 	fi
 	texi2dvi --pdf --batch xforms.texi
