--- origsrc/xforms-1.2.4/doc/Makefile.am	2013-12-19 06:21:15.000000000 -0500
+++ src/xforms-1.2.4/doc/Makefile.am	2017-06-10 21:51:32.881871500 -0400
@@ -92,7 +92,7 @@ uninstall-local:
 	cp -r xforms_images xforms.html
 
 .texi.pdf:
-	if [ $(TEXI2DVI_NEEDS_PDF) = "yes" ]; then \
+	if [ x$(TEXI2DVI_NEEDS_PDF) = "xyes" ]; then \
 		$(MAKE) -C xforms_images create_pdf_files; \
 	fi
 	texi2dvi --pdf --batch xforms.texi
--- origsrc/xforms-1.2.4/fdesign/Makefile.am	2014-06-28 16:34:29.000000000 -0400
+++ src/xforms-1.2.4/fdesign/Makefile.am	2017-06-10 21:29:25.023089200 -0400
@@ -15,7 +15,7 @@ INCLUDES = -I./xpm $(X_CFLAGS)
 
 bin_PROGRAMS = fdesign
 
-fdesign_LDADD = $(top_srcdir)/lib/libforms.la $(X_LIBS) $(X_PRE_LIBS) $(XPM_LIB) -lX11 $(LIBS) $(X_EXTRA_LIBS)
+fdesign_LDADD = $(top_builddir)/lib/libforms.la $(X_LIBS) $(X_PRE_LIBS) $(XPM_LIB) -lX11 $(LIBS) $(X_EXTRA_LIBS)
 
 fdesign_SOURCES = \
 	fd_attribs.c \
